### DISCUSSION TITLE
Unable to activate upload option in Photoprism settings

### DISCUSSION THREAD

**Discussion by Mike-Sundays** [photoprism/photoprism#3070] [Q&A]
Hi everyone!

I am trying to setup Photoprism on my raspberry pi. I am using the recommended docker-compose file and the read-only mode is disabled, but it seems that the Upload and Import checkboxes are greyed out, and cannot be toggled on or off. I have searched the documentation and github for this particular issue, but I was not able to find any answer. Could it be a permissions issue?

Thanks!

**Comment by rosscdh** [comment]
is still an issue

**Reply by lastzero** [reply]
Please check the logs and make sure the filesystem / folders are writable by the photoprism service, especially if you don't run it as root.
